tests/presubmit/presubmit.sh

```diff
@@ -9,6 +9,25 @@ sudo chmod +x "$KOKORO_GFILE_DIR/verify-commits.sh"
 sudo cp "$KOKORO_GFILE_DIR"/shellcheck-latest.linux /usr/local/bin/shellcheck
 sudo chmod +x /usr/local/bin/shellcheck
 
+# Maybe grab and configure the bazel cacher
+if [ -e "$KOKORO_GFILE_DIR"/bazel-cache ]; then
```

Comment:
When is this file present:
"$KOKORO_GFILE_DIR"/bazel-cache

Reply:
Sorry, going to drop this PR and handle this in the kokoro configs. The file should only have been present for debian-docker presubmit builds before this, as a test.